messages.pot

```diff
@@ -0,0 +1,79 @@
+# Translations template for PROJECT.
```

Comment:
Could this file be moved out of the project root ? Maybe inside `app/translations` ?